lib/widgets/CustomButton/custom_button.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:classroom_app/styles/color/colors.dart';
+
+class CustomButton extends StatelessWidget {
+  const CustomButton({
+    super.key,
+    required this.buttonText,
+    required this.buttonBgColor,
+    this.buttonTextColor,
+    this.buttonBorderColor,
+  });
+
+  final String buttonText;
+  final Color buttonBgColor;
+  final Color? buttonTextColor;
+  final Color? buttonBorderColor;
+
+  @override
+  Widget build(BuildContext context) {
+    double screenWidth = MediaQuery.of(context).size.width;
+    return SizedBox(
+      height: 56,
```

Comment:
The button height is fixed at 56. This might not look good on smaller or larger screens.
Make the height dynamic based on screen size or device type.

```suggestion
      height: screenWidth * 0.14,
```

Reply:
not using anymore